crates/bitwarden/src/client/client.rs

```diff
@@ -132,6 +132,11 @@ impl Client {
         &self.__api_configurations
     }
 
+    #[cfg(feature = "mobile")]
+    pub(crate) fn get_http_client(&self) -> &reqwest::Client {
+        &self.__api_configurations.api.client
```

Comment:
Do we want to expose the api configurations http client? I think it pre-sets some headers that may be sensitive.

Reply:
We don't really set any headers on the client at the moment:
https://github.com/bitwarden/sdk/blob/7ec2183ab8194a05c994c9c67909dec538da1096/crates/bitwarden/src/client/client.rs#L87-L92

We probably had some in the past, because as it is, that `headers` map is not very useful. At the moment all the other headers are added to each request individually. We also have cookies disabled on the client so I think it should be reasonably safe.